src/components/flow/actions/startsession/__snapshots__/StartSessionForm.test.tsx.snap

```diff
@@ -935,6 +937,7 @@ Array [
     },
     "flow": Object {
       "name": "My Flow",
+      "type": "flow",
```

Comment:
this feels like a hint that something with my testing approach is off, as here i think the type should either be "messaging" or "voice" and not "flow

Reply:
unless this might be ok as it looks like this is just a flowContext.ts -> AssetType.Flow enum -> "flow"

Reply:
If memory serves this is for the generic asset selection stuff. It can work against flows, contacts, groups, etc.